src/makeCalculator.js

```diff
@@ -37,7 +37,27 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (result, number) => result + number,
```

Comment:
The 'add', 'subtract', 'multiply', 'divide' methods should not receive the 'result' as a parameter. They should use 'this.result' instead.